<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7SM6G84" failures="0" tests="6" name="CreateOrganization_inRealTime_byUsingBaseClass.OrganizationCreationByUsingBaseClass" time="59.829" errors="1" timestamp="2024-11-04T16:08:57 IST" skipped="5">
  <testcase classname="CreateOrganization_inRealTime_byUsingBaseClass.OrganizationCreationByUsingBaseClass" name="CreateOrgTest" time="9.212">
    <error message="Cannot invoke &quot;Object.toString()&quot; because the return value of &quot;org.apache.poi.ss.usermodel.Row.getCell(int)&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "Object.toString()" because the return value of "org.apache.poi.ss.usermodel.Row.getCell(int)" is null
at Com.comcast.crm.genaric.fileutility.ExcelUtility.getDataFromTheExcel(ExcelUtility.java:22)
at CreateOrganization_inRealTime_byUsingBaseClass.OrganizationCreationByUsingBaseClass.CreateOrgTest(OrganizationCreationByUsingBaseClass.java:52)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- CreateOrgTest -->
  <system-out/>
  <testcase classname="CreateOrganization_inRealTime_byUsingBaseClass.OrganizationCreationByUsingBaseClass" name="CreateOrgTest" time="9.377">
    <skipped/>
  </testcase> <!-- CreateOrgTest -->
  <system-out/>
  <testcase classname="CreateOrganization_inRealTime_byUsingBaseClass.OrganizationCreationByUsingBaseClass" name="CreateOrgTest" time="10.956">
    <skipped/>
  </testcase> <!-- CreateOrgTest -->
  <system-out/>
  <testcase classname="CreateOrganization_inRealTime_byUsingBaseClass.OrganizationCreationByUsingBaseClass" name="CreateOrgTest" time="10.474">
    <skipped/>
  </testcase> <!-- CreateOrgTest -->
  <system-out/>
  <testcase classname="CreateOrganization_inRealTime_byUsingBaseClass.OrganizationCreationByUsingBaseClass" name="CreateOrgTest" time="8.255">
    <skipped/>
  </testcase> <!-- CreateOrgTest -->
  <system-out/>
  <testcase classname="CreateOrganization_inRealTime_byUsingBaseClass.OrganizationCreationByUsingBaseClass" name="CreateOrgTest" time="11.555">
    <skipped/>
  </testcase> <!-- CreateOrgTest -->
  <system-out/>
</testsuite> <!-- CreateOrganization_inRealTime_byUsingBaseClass.OrganizationCreationByUsingBaseClass -->
